detection/impact/evenly-timestomped.sql

```diff
@@ -47,6 +47,7 @@ WHERE
     'printcap',
     'strace-log-merge',
     'installer-info.json'
+    'debian-keyring.gpg'
```

Comment:
Instead of this, what do you think about removing `/usr` from the list of directories this query scans?

I don't really care about anything in `/usr/share`; just libraries, config files, and binary directories. Modern Linux distros typically symlink /bin and /sbin to /usr/[,s]bin anyways.